xandikos/web.py

```diff
@@ -1148,6 +1152,28 @@ def get_current_user_principal(env):
             ]
         )
 
+    async def _handle_request(self, request, environ, start_response=None):
+        # don't look any deeper than /user/calendars/foobar/.git
+        path_search_prefix = request.path.split(posixpath.sep)[:5];
```

Comment:
Some of the style checks appear to be failing.

Reply:
fixed `flake8`, `isort` seems to be red anyway?!